src/platform_impl/linux/wayland/event_loop.rs

```diff
@@ -282,8 +282,14 @@ impl<T: 'static> Clone for EventLoopProxy<T> {
 }
 
 impl<T: 'static> EventLoopProxy<T> {
-    pub fn send_event(&self, event: T) -> Result<(), EventLoopClosed> {
-        self.user_sender.send(event).map_err(|_| EventLoopClosed)
+    pub fn send_event(&self, event: T) -> Result<(), EventLoopClosed<T>> {
+        self.user_sender.send(event).map_err(|e| {
+            EventLoopClosed(if let ::calloop::channel::SendError::Disconnected(x) = e {
+                x
+            } else {
+                unreachable!()
```

Comment:
nit: should still return `Err` and not panic. Same for x11.

Reply:
https://github.com/rust-windowing/winit/pull/1294#discussion_r352815218

Reply:
https://twitter.com/ZelleTTY/status/1202678011981697027

Reply:
https://github.com/rust-windowing/winit/pull/1294#discussion_r352815218

Reply:
How? This was already considered and dismissed.

Reply:
Ah, I did not notice. Apologies.